### PR TITLE
Customize action buttons in admin class

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

**Comment by OskarStark** [comment]
awesome work @core23 

could you add a test for the new methods?

**Comment by core23** [comment]
Added a basic test for the new method. 

Sadly, I wasn't able to add a test for the ``AdminExtension`` because the ``method_exists`` check doesn't work in phpunit.

**Comment by OskarStark** [comment]
Thank you very much, can you please add a new PR with an UPgrade Notice for the next major?

Thank you @core23

**Comment by tim96** [comment]
Awesome idea, thank you @core23

File: Admin/Admin.php [hunk 1]
```diff
@@ -2955,4 +2955,70 @@ public function checkAccess($action, $object = null)
...
+     *
+     * @return array
+     */
+    public function getActionButtons($action, $object = null)
+    {
+        $list = $this->configureActionButtons($action, $object);
+
+        foreach ($this->getExtensions() as $extension) {
+            // TODO: remove method check in next major release
+            if (method_exists($extension, 'configureActionButtons')) {
```
Comment (OskarStark) [inline Admin/Admin.php]: @core23 can you create a PR for the next major please?

**Pull request merged**